### PR TITLE
Add footer for articles

### PR DESCRIPTION
**Pull request by gitsobek**

### CODE REVIEW

File: components/partials/ArticleFooter.vue [hunk 1]
```diff
@@ -1,17 +1,213 @@
...
 
-</style>
+.article-footer {
+  $p: &;
+  padding: 0 30px 30px 30px;
+  height: 250px;
+  background-color: $article-footer-bg-color;
+
+  &__tags {
+    list-style: none;
```
Comment (FieryCod) [inline components/partials/ArticleFooter.vue]: indent

File: components/partials/ArticleFooter.vue [hunk 2]
```diff
@@ -1,17 +1,213 @@
...
+    position: relative;
+    font-size: 0.80rem;
+  }
+
+  &__single-tag-route {
+    background: $gray-20;
+    color: $gray-50;
+    display: block;
+    line-height: 1em;
+    padding: 0.65em 1.5em 0.65em 1.5em;
```
Comment (FieryCod) [inline components/partials/ArticleFooter.vue]: .65

File: components/partials/ArticleFooter.vue [hunk 3]
```diff
@@ -1,17 +1,213 @@
...
+    margin-right: 5px;
+    margin-left: 5px;
+    position: relative;
+  }
+
+  &__single-member-tag-route {
+    color: $gray-10;
+    font-weight: bold;
+    font-size: 0.7rem;
+    display: block;
```
Comment (FieryCod) [inline components/partials/ArticleFooter.vue]: indent

File: components/partials/ArticleFooter.vue [hunk 4]
```diff
@@ -1,17 +1,213 @@
...
+    position: relative;
+    left: 50px;
+  }
+
+  &__single-icon {
+    font-size: 0.9rem;
+    color: $gray-40;
+    margin: 0 5px;
+
+    &:before{
```
Comment (FieryCod) [inline components/partials/ArticleFooter.vue]: indent

File: components/partials/ArticleFooter.vue [hunk 5]
```diff
@@ -1,17 +1,213 @@
...
+    display: inline-block;
+    margin-right: 5px;
+    margin-left: 5px;
+    position: relative;
+  }
+
+  &__single-member-tag-route {
+    color: $gray-10;
+    font-weight: bold;
+    font-size: 0.7rem;
```
Comment (FieryCod) [inline components/partials/ArticleFooter.vue]: .7rem; not 0.7

File: components/partials/ArticleFooter.vue [hunk 6]
```diff
@@ -1,17 +1,213 @@
...
+  }
+
+  &__member-tags {
+    list-style: none;
+  }
+
+  &__single-member-tag {
+    display: inline-block;
+    margin-right: 5px;
+    margin-left: 5px;
```
Comment (FieryCod) [inline components/partials/ArticleFooter.vue]: use shorthand there

File: components/partials/ArticleFooter.vue [hunk 7]
```diff
@@ -1,17 +1,213 @@
...
+    font-size: 0.80rem;
+  }
+
+  &__leader-tags {
+    list-style: none;
+  }
+
+  &__single-leader-tag {
+    display: inline-block;
+    margin-right: 10px;
```
Comment (FieryCod) [inline components/partials/ArticleFooter.vue]: use shorthand

File: components/partials/ArticleFooter.vue [hunk 8]
```diff
@@ -1,17 +1,213 @@
...
 
+  &__author-member-sections {
+    font-size: 0.80rem;
+  }
+
+  &__leader-tags {
+    list-style: none;
+  }
+
+  &__single-leader-tag {
```
Comment (FieryCod) [inline components/partials/ArticleFooter.vue]: Please refactor it, a tag class can be made there.

File: components/partials/ArticleFooter.vue [hunk 9]
```diff
@@ -1,17 +1,213 @@
...
+  &__author-description {
+    width: 50%;
+    padding: 10px 0;
+    font-size: 0.80rem;
+    color: $gray-40;
+  }
 
+  &__author-leader-sections {
+    padding: 10px 0;
+    font-size: 0.80rem;
```
Comment (FieryCod) [inline components/partials/ArticleFooter.vue]: or even .8rem

File: components/partials/ArticleFooter.vue [hunk 10]
```diff
@@ -1,17 +1,213 @@
...
+  props: articleFooterSchema.props
 }
 </script>
 
 <style lang="scss">
+@import "assets/scss/_imports.scss";
 
-</style>
+.article-footer {
+  $p: &;
```
Comment (FieryCod) [inline components/partials/ArticleFooter.vue]: its unused please delete t

File: components/partials/ArticleFooter.vue [hunk 11]
```diff
@@ -1,17 +1,194 @@
...
-  <footer style="height: 200px; background-color: lightgray;">
-    Tags, about author and social buttons
+  <footer class="article-footer">
+    <ul class="article-footer__tags">
+      <li v-for="(tag, index) in tags" class="article-footer__single-tag" :key = "index">
+        <router-link to="" class="article-footer__single-tag-route"> {{ tag }} </router-link>
+      </li>
+    </ul>
+    <div class="article-footer__footer-box">
+      <img class="article-footer__author-avatar" :src="author.avatar.url"/>
```
Comment (zrebcu411) [inline components/partials/ArticleFooter.vue]: Avatar should link to the author's profile. Please wrap it in a `router-link` tag. Add also `:alt="author.fullname"` attribute.

File: components/partials/ArticleFooter.vue [hunk 12]
```diff
@@ -1,17 +1,194 @@
...
+      <div class="article-footer__author-details">
+        <span class="article-footer__author-fullname article-footer__details-text"> {{ author.fullname }} </span>
```
Comment (zrebcu411) [inline components/partials/ArticleFooter.vue]: Same as above - should link to the profile.

File: components/partials/ArticleFooter.vue [hunk 13]
```diff
@@ -1,17 +1,194 @@
...
+  &__tags {
+    list-style: none;
+    padding-bottom: 20px;
+  }
+
+  &__single-tag {
+    display: inline-block;
+    margin-right: 16px;
+    position: relative;
+    font-size: .8rem;
```
Comment (zrebcu411) [inline components/partials/ArticleFooter.vue]: Please add hover and focus effect like in article card on the homepage. Just lighter gray color will be ok.

File: components/partials/ArticleFooter.vue [hunk 14]
```diff
@@ -1,17 +1,194 @@
...
+    font-weight: bold;
+    font-size: .7rem;
+    display: block;
+    line-height: .6em;
+    padding: .6em 1em .6em 1em;
+    text-decoration: none;
+    border-radius: 30px;
+  }
+
+  &__icons {
```
Comment (zrebcu411) [inline components/partials/ArticleFooter.vue]: Same as above. Hover and focus effect like in article cards on the homepage.

File: schemes/article/footer.js [hunk 15]
```diff
@@ -0,0 +1,16 @@
+export default {
+  props: {
+    likes: {
+      type: Array,
+      default: () => []
+    },
+    comments: {
```
Comment (zrebcu411) [inline schemes/article/footer.js]: Likes and comments should be a number, not an array. I'm not sure but, every article has commentsCount property.

**Comment by zrebcu411** [comment]
I have made some comments. Very good job Piotrek!

**Pull request merged**